src/test/java/BaseTest.java

```diff
@@ -10,22 +11,29 @@ public class BaseTest {
 
 
     WebDriver driver;
-    String url;
+
+    String url = "https://bbb.testpro.io/";
 
     @BeforeSuite
     public static void chromeConfigs() {
         // This is for Windows users
         if (System.getProperty("os.name").toLowerCase().contains("win")) {
             System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+
+
         }
     }
 
     @BeforeMethod
     public void launchBrowser() {
-        driver = new ChromeDriver();
+        ChromeOptions chromeOptions = new ChromeOptions();
+        chromeOptions.addArguments("--disable-notifications");
+
+        driver = new ChromeDriver(chromeOptions);
```

Comment:
You can add `driver.manage().window().maximize();`

![image](https://user-images.githubusercontent.com/104761482/198562077-4c9b7ecf-b4c0-4502-b22c-8aa954666718.png)